<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditSchedulersClass</class>
 <widget class="QDialog" name="EditSchedulersClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>291</width>
    <height>482</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>291</width>
    <height>482</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>EditSchedulers</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="lblAction">
        <property name="text">
         <string>Action</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="4">
       <widget class="QComboBox" name="cmbAction">
        <item>
         <property name="text">
          <string>StartNew</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Resume</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Stop</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblDuration">
        <property name="text">
         <string>Duration</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="4">
       <widget class="QComboBox" name="cmbDuration">
        <item>
         <property name="text">
          <string>2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>30</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>60</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>90</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>120</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lblRepeat">
        <property name="text">
         <string>Repeat</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="4">
       <widget class="QComboBox" name="cmbRepeat">
        <item>
         <property name="text">
          <string>Once</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Daily</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Weekly</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="lblDate">
        <property name="text">
         <string>Date</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="2">
       <widget class="QDateEdit" name="dateEdit">
        <property name="date">
         <date>
          <year>2012</year>
          <month>1</month>
          <day>1</day>
         </date>
        </property>
       </widget>
      </item>
      <item row="4" column="4">
       <widget class="QTimeEdit" name="timeEdit">
        <property name="dateTime">
         <datetime>
          <hour>0</hour>
          <minute>0</minute>
          <second>0</second>
          <year>2012</year>
          <month>1</month>
          <day>1</day>
         </datetime>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QLabel" name="lblTime">
        <property name="text">
         <string>Time</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="weeklyFrame">
     <property name="title">
      <string>DAYS</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QCheckBox" name="chkSunday">
        <property name="text">
         <string>SUN</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="chkMonday">
        <property name="text">
         <string>MON</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="chkTuesday">
        <property name="text">
         <string>TUES</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="chkWenesday">
        <property name="text">
         <string>WEDNES</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="chkThursday">
        <property name="text">
         <string>THUR</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QCheckBox" name="chkFriday">
        <property name="text">
         <string>FRI</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="chkSaturday">
        <property name="text">
         <string>SAT</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="2" column="0">
       <widget class="QLabel" name="lblEventNo">
        <property name="text">
         <string>Event No</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="lblName">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="lblDescription">
        <property name="text">
         <string>Description</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lblEventType">
        <property name="text">
         <string>EventType</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2" colspan="3">
       <widget class="QComboBox" name="cmbEventType"/>
      </item>
      <item row="4" column="2" colspan="3">
       <widget class="QLineEdit" name="txtName"/>
      </item>
      <item row="5" column="2" colspan="3">
       <widget class="QLineEdit" name="txtDescription"/>
      </item>
      <item row="6" column="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="6" column="3">
       <widget class="QPushButton" name="btnOk">
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
      <item row="6" column="4">
       <widget class="QPushButton" name="btnCancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" colspan="3">
       <widget class="QComboBox" name="cmbEventNo">
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2" colspan="3">
       <widget class="QComboBox" name="cmbLocation"/>
      </item>
      <item row="1" column="2" colspan="3">
       <widget class="QComboBox" name="cmbRoom"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Location</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Room</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>EditSchedulersClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>265</x>
     <y>453</y>
    </hint>
    <hint type="destinationlabel">
     <x>288</x>
     <y>455</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
